src/app/donation-complete/donation-complete.component.html

```diff
@@ -28,7 +28,15 @@
       <div class="thank-you">
         <h2>Thank you!</h2>
 
-        <p class="thank-you-text">Your generous donation has been made successfully</p>
+        <p class="thank-you-text">
+          Your generous donation of {{ donation.donationAmount | exactCurrency:donation.currencyCode }} has been made.
+          That’s worth {{ totalValue | exactCurrency:donation.currencyCode }} to {{ donation.charityName }}.
```

Comment:
changed `,` to `.` at end here. I think `.` is probably better anyway, and it's simpler as the next words have to be conditional on how the donation was paid.